src/main/java/com/revature/ams/Member/MemberController.java

```diff
@@ -1,4 +1,89 @@
 package com.revature.ams.Member;
 
+// TODO: REVIEW ME
+
+import java.util.Scanner;
+
 public class MemberController {
+    private final Scanner scanner;
+    private final MemberService memberService;
+    private int count = 0;
+
+    public MemberController(Scanner scanner, MemberService memberService) {
+        this.scanner = scanner;
+        this.memberService = memberService;
+    }
+
+    public void register(){
+        System.out.println("Please enter first name: ");
+        String firstName = scanner.next();
+
+        System.out.println("Please enter last name: ");
+        String lastName = scanner.next();
+
+        System.out.println("Please enter email: ");
+        String email = scanner.next();
+
+        Member.MemberType memberType = Member.MemberType.valueOf("PASSENGER");
```

Comment:
Can you please explain why the member type here is Passenger and not one of the other types

Reply:
Absolutely, by default we want members to be memberType PASSENGER, anyone who is an admin or a pilot will be given credentials for security purposes. There are a few alternative ways to default this information as well.